lib/usecases/run/EndRunUseCase.js

```diff
@@ -26,10 +26,13 @@ class EndRunUseCase {
     async execute(dto) {
         const { body, params } = dto;
 
+        const { lhcPeriod } = body;
+        delete body.lhcPeriod;
+
         // Bad variable name, the parameter expect a run number
         const { runId: runNumber } = params;
         try {
-            return { result: await runService.update({ runNumber }, { runPatch: body }) };
+            return { result: await runService.update({ runNumber }, { runPatch: body, relations: { lhcPeriod } }) };
```

Comment:
Here, I think you should put lhcPeriodName
If you did not catch it, it means that a test should be added